src/components/DisplayInput.js

```diff
@@ -84,20 +93,20 @@ const DisplayInput = React.createClass({
         </div>
 
         <Row>
```

Comment:
Having `status` on the new row makes it more flexible for `small`, `medium`, and `large` views. 👍